### ISSUE TITLE
Feature Request : Italian Translation

### ISSUE THREAD

**Issue by hyunkel76** [fafranco82/swdestinydb#66]
Hi there
Great work on SWdestinydb.
I'm interested to help in Italian translation, but i'm totally new to this platform.
In the interest of Italian community i really would like to help.

**Comment by fafranco82** [comment]
There is already a thread talking about Italian translation (#46). I've just asked him how much progress did him.

**Comment by KlarkD** [comment]
Actually i'm focusing on traslating the cards (json files). This will be done in a few days, once i get original italian translation. Then i will focus on translating the site. As i can see it's a bit more tricky...still learning about some programming languages that could be helpful for the project.

**Comment by fafranco82** [comment]
I've changed the way the site can be translated. Now it's easier (and don't need to install a local copy if you don't want). You can see the new instructions in at the end of the README.

**Comment by fafranco82** [comment]
@KlarkD, @hyunkel76 , if any of  you are interested in translating the website, you can send me an email to webmaster@swdestinydb.com providing your email and name, to send you back an invitation to the translation web platform :)

**Comment by KlarkD** [comment]
Thank you, email sent.
BTW, IT-traslation on json files is done. I pulled a request from my forked branch. Is this the right way to say 'work is done'?
@fafranco82 A validation check should be done on them, though. Not able to do it myself, atm. So could you handle it, please?

**Comment by fafranco82** [comment]
Yes, to say "work is done" you make a pull request. But... I don't see any pull request on swdestinydb-json-data :(

**Comment by fafranco82** [comment]
http://it.swdestinydb.com/set/AW/spoiler

Cards is up in the web :)
